README.md

```diff
@@ -6,12 +6,50 @@ Amido Stacks targets different cloud providers.
 
 [Azure](https://amido.github.io/stacks/docs/workloads/azure/backend/netcore/introduction_netcore)
 
+## Folders of interest in this repository
+
+```shell
+stacks-dotnet-cqrs-events
+│   README.md
+└─── src
+│    └─── api
+│    │    └─── xxAMIDOxx.xxSTACKSxx.API
+│    │    └─── other API related projects
+│    └─── functions
+│    │    └─── func-aeh-listener
+│    │         └─── xxAMIDOxx.xxSTACKSxx.Listener
+│    │    └─── func-asb-listener
+│    │         └─── xxAMIDOxx.xxSTACKSxx.Listener
+│    │    └─── func-cosmosdb-worker
+│    │         └─── xxAMIDOxx.xxSTACKSxx.Worker
+│    └─── tests
+│    └─── worker
+│         └─── xxAMIDOxx.xxSTACKSxx.BackgroundWorker
+```
+
+- The `api` folder contains everything related to the API and is a standalone executable
+- The `functions` folder contains 3 sub-folders with Azure Functions solutions
+  - `func-aeh-listener` is an Azure Event Hub trigger that listens for `MenuCreatedEvent`
+  - `func-asb-listener` is an Azure Service Bus subscription (filtered) trigger that listens for `MenuCreatedEvent`
+  - `func-cosmosdb-worker` is a CosmosDB change feed trigger function that publishes a `CosmosDbChangeFeedEvent` when a new entity has been added or was changed to CosmosDB
+- The `worker` folder contains a background worker that listens to all event types from the ASB topic and shows example handlers for them and the use of the [Amido.Stacks.Messaging.Azure.ServiceBus](https://github.com/amido/stacks-dotnet-packages-messaging-asb) package.
+
+The API, functions and worker all depend on the [Amido.Stacks.Messaging.Azure.ServiceBus](https://github.com/amido/stacks-dotnet-packages-messaging-asb),  the [Amido.Stacks.Messaging.Azure.EventHub](https://github.com/amido/stacks-dotnet-packages-messaging-aeh) or the [Amido.Stacks.SQS](https://github.com/amido/stacks-dotnet-packages-sqs) packages for their communication with either Azure Service Bus, Azure Event Hub or AWS SQS depending on the specific implementation.
+
+The functions and workers are all stand-alone implementations that can be used together or separately in different projects.
+
 ## Templates
 
 All templates from this repository come as part of the [Amido.Stacks.CQRS.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/) NuGet package. The list of templates inside the package are as follows:
 
 - `stacks-cqrs-webapi`. The full CQRS template including source + build infrastructure.
 - `stacks-add-cqrs`. A special template that can add `CQRS` functionality and projects to your existing Web API solution
+- `stacks-cqrs-events-app`. The full template including source + build infrastructure.
```

Comment:
This will have to be changed to reflect the name change on line 15 in the template.json file.